lib/ui/common/MyPainter.dart

```diff
@@ -0,0 +1,53 @@
+
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+class MyPainter extends CustomPainter{
```

Comment:
very poor name. What does it do?

Reply:
File deleted, now it's a private class in the CircularWidget file like we talked

Reply:
👍